lib/src/main/java/com/auth0/android/lock/LockCallback.java

```diff
@@ -68,5 +69,5 @@
      *
      * @param error describing what happened.
      */
-    void onError(@NonNull LockException error);
+    void onError(@NonNull AuthenticationException error);
```

Comment:
Changes the type of exception. Helps to deliver the actual exception and not wrapping everything again.